sdk/monitor/azure-monitor-opentelemetry/azure/monitor/opentelemetry/_configure.py

```diff
@@ -153,6 +172,10 @@ def _setup_metrics(configurations: Dict[str, ConfigurationValue]):
     set_meter_provider(meter_provider)
 
 
+def _setup_live_metrics(configurations):
+    enable_live_metrics(**configurations)
```

Comment:
Nit: If we don't expect anything else in this function in the future, let's just call enable_live_metrics at the higher level